server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -36,6 +36,24 @@ Feature: GetMessages method
     And the id of the message is "m1"
     And the message is in "custom,INBOX" mailboxes
 
+  Scenario: Retrieving a message in a mailbox delegated to me
+    Given "alice@domain.tld" has a mailbox "shared"
+    Given "alice@domain.tld" shares its mailbox "shared" with "bob@domain.tld"
+    And the user has a message "m1" in "shared" mailbox with subject "my test subject", content "testmail"
+    Given "bob@domain.tld" is connected
+    When the user ask for messages "m1"
+    Then no error is returned
+    And the list should contain 1 message
+    And the id of the message is "m1"
+
+  Scenario: Retrieving a message in a mailbox not delegated to me
+    Given "alice@domain.tld" has a mailbox "notShared"
+    And the user has a message "m1" in "notShared" mailbox with subject "my test subject", content "testmail"
+    Given "bob@domain.tld" is connected
+    When the user ask for messages "m1"
+    Then no error is returned
+    And the list should contain 0 message
+
```

Comment:
Can you add:
Scenario: Retrieving a message in a mailbox delegated to someone else